openedx/core/djangoapps/coursegraph/tasks.py

```diff
@@ -12,8 +12,8 @@
 from edx_django_utils.cache import RequestCache
 from edx_django_utils.monitoring import set_code_owner_attribute
 from opaque_keys.edx.keys import CourseKey
-from py2neo import Graph, Node, Relationship, authenticate, NodeSelector
-from py2neo.compat import integer, string
+from py2neo import Graph, Node, Relationship, NodeMatcher
+from py2neo.compat import integer_types, string_types
```

Comment:
This [PR](https://github.com/technige/py2neo/commit/0cda4fcbaee6b249ed4a9044188f661e46b93626#diff-5e882fcc90e074f92d914af7c77b810929459dfa8294587c96e945ced6ad7c6bR39) shows this change